CONTRIBUTING.md

```diff
@@ -89,6 +157,45 @@ We welcome contributions, large or small, including:
 - Example code snippets
 - Sample data
 
-If you would like to contribute new APIs, please check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file to ensure your contributions align with the package owner's and project goals.
+If you would like to contribute new APIs, create a new [discussions post](https://github.com/iTwin/itwinjs-core/discussions) and explain what these new APIs aim to accomplish. If possible check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file and tag the owners of the package you plan on introducing the new APIs into.
```

Comment:
shouldnt we push new feature reqs to issues?
i feel discussions are for more quick/formal things

Reply:
I do notice many requests in other OSS projects come from the issues tab too... okay edited to suggest opening an issue instead.